pkg/restapi/operation/operations.go

```diff
@@ -336,6 +339,34 @@ func (c *Operation) updateDocumentHandler(rw http.ResponseWriter, req *http.Requ
 	c.updateDocument(rw, requestBody, docID, vaultID)
 }
 
+// Delete Document swagger:route DELETE /encrypted-data-vaults/{vaultID}/documents/{docID} deleteDocumentReq
+//
+// Delete an encrypted document.
+//
+// Responses:
+//		default: genericError
+// 			200: emptyRes
+//			400: emptyRes
+// 			404: emptyRes
+func (c *Operation) deleteDocumentHandler(rw http.ResponseWriter, req *http.Request) {
+	vaultID, success := unescapePathVar(vaultIDPathVariable, mux.Vars(req), rw)
+	if !success {
+		return
```

Comment:
add log statement here to help troubleshoot this case

Reply:
To me this seems like it should return an error.

Reply:
`unescapePathVar` was where the response was written. I have changed that to log the error and return BadRequest:

https://github.com/trustbloc/edv/pull/137/files#diff-5113c7f3feb6ad875b4dad132a94e674a7dbe720f0fa70d2b8ecee5c630191c3R24